### PR TITLE
Fix markdown -> html conversion Math symbol breaking issues

### LINKED ISSUE #287: 新版网站-深度学习入门-机器翻译页面下版面有问题
1、有的公式未被浏览器正确显示出来
2、“、、、”

ppp

### PR DESCRIPTION
**Pull request by varunarora**
Fixes #287

### CODE REVIEW

File: portal/deploy/documentation_generator.py [hunk 1]
```diff
@@ -18,6 +18,19 @@
 ]
 
 
+def enhance_markdown(markdown_body):
+    """
```
Comment (jetfuel) [inline portal/deploy/documentation_generator.py]: awesome description.

File: portal/deploy/documentation_generator.py [hunk 2]
```diff
@@ -18,6 +18,19 @@
 ]
 
 
+def enhance_markdown(markdown_body):
```
Comment (cs2be) [inline portal/deploy/documentation_generator.py]: maybe sanatize_markdown makes more sense

File: portal/deploy/documentation_generator.py [hunk 3]
```diff
@@ -18,6 +18,19 @@
...
+    """
+    There are some symbols used in the markdown body, which when go through Markdown -> HTML
+    conversion, break. This does a global replace on markdown strings for these symbols.
+    """
+    return markdown_body.replace(
+        # This is to solve the issue where <s> and <e> are interpreted as HTML tags
+        '\<s>', '&lt;s&gt;').replace(
+        '\<e>', '&lt;e&gt;').replace(
+        '&lt;', '<').replace(
+        '&gt;', '>')
```
Comment (jetfuel) [inline portal/deploy/documentation_generator.py]: So we basically replace `\<s>` to `<s>` and `\<e>` to `<e>`
why don't we just do that directly?

Comment (varunarora) [reply portal/deploy/documentation_generator.py]: What do you mean by directly?

File: portal/deploy/documentation_generator.py (hunk 2 above)
Comment (varunarora) [reply portal/deploy/documentation_generator.py]: *sanitize, yes :)

File: portal/deploy/documentation_generator.py (hunk 3 above)
Comment (jetfuel) [reply portal/deploy/documentation_generator.py]: Like this. `markdown_body.replace('\<s>', '<s>')`

**Pull request merged**